### ISSUE TITLE
extract 11ty customisations into NHSBSA 'theme'

### ISSUE THREAD

**Issue by yantantether** [nhsbsa/nhsbsa-digital-playbook#122]
## What
11ty doesn't support themes, but it should be possible to extract all the customisations we've done in the playbook, into a library.

## Why
Some groups in the BSA want to maintain their own open source, NHSBSA branded documentation.
Extracting the work done for the playbook will make this easier to maintain a common approach to sites like this one.

**Comment by WORTI3** [comment]
### Discussed in https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/209

<div type='discussions-op-text'>

<sup>Originally posted by **WORTI3** August 30, 2023</sup>
### What
Extract common code to the [nhsbsa-docs](https://github.com/nhsbsa/nhsbsa-docs) library.

### Why
The playbook acts as a handbook that others can easily contribute to and fork for other information-based playbooks. All boilerplate and components that aren't playbook-specific should be moved to the nhsbsa-docs repo, as this is technical and won't be touched by those adding pages. 

In short: this will improve the ability to add new content to the playbook and simplify the code involved. 

We should be able to install the NHSBSA-docs package into the playbook to gain access to specific functionality as needed. 

### Anything else
Example code available on WIP branches.

</div>
